From 69c89cb453c0beac5d8349108cee8f6806e5db19 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Tue, 17 May 2016 05:49:17 +0200
Subject: [PATCH 100/102] MAINTAINERS: add Sean as mediatek ethernet
 maintainer

Sean has been busy doing most of the QA and stress testing of the driver.
Add him to the list of maintainers.

Signed-off-by: Sean Wang <keyhaede@gmail.com>
Signed-off-by: John Crispin <john@phrozen.org>
---
 MAINTAINERS |    1 +
 1 file changed, 1 insertion(+)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6905,6 +6905,7 @@ F:	include/uapi/linux/uvcvideo.h
 MEDIATEK ETHERNET DRIVER
 M:	Felix Fietkau <nbd@nbd.name>
 M:	John Crispin <john@phrozen.org>
+M:	Sean Wang <keyhaede@gmail.com>
 L:	netdev@vger.kernel.org
 S:	Maintained
 F:	drivers/net/ethernet/mediatek/
